data/hai missions.txt

```diff
@@ -1053,3 +1053,245 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Nanachi 1"
+	name "Escort Nanachi to <planet>"
+	description "Help Nanachi, a Hai who just bought her first ship, make a simple delivery to Cloudfire."
+	minor
+	source "Giverstone"
+	destination "Cloudfire"
+	deadline
+	to offer
+		"net worth" > 20000000
+		"combat rating" > 1200
+		year > 3018
+	on offer
+		conversation
+			`	As you are making your way through the spaceport you notice a worried-looking Hai staring at the local job board and back at some sort of document they are holding.`
```

Comment:
The first line in a conversation doesn't get tabbed in.
```suggestion
			`As you are making your way through the spaceport you notice a worried-looking Hai staring at the local job board and back at some sort of document they are holding.`
```
This needs changed throughout.